packages/types/src/signUp.ts

```diff
@@ -114,6 +115,12 @@ export type PrepareVerificationParams =
       strategy: OAuthStrategy;
       redirectUrl?: string;
       actionCompleteRedirectUrl?: string;
+    }
+  | {
+      strategy: SamlStrategy;
+      samlIdentifier: string;
+      redirectUrl?: string;
+      actionCompleteRedirectUrl?: string;
     };
```

Comment:
❓ isn't this like `SamlConfig` type extended with a `samlIdentifier`?

Reply:
Yes, but since we're not DRYing this up for the oauth case either, I preferred to be consistent.